### ISSUE TITLE
nanopolish variants --consensus argument

### ISSUE THREAD

**Issue by igordot** [jts/nanopolish#94]
I am trying to understand the documentation. In the README, the example is:
```
nanopolish variants --consensus polished.{1}.fa -w {1} -r reads.fa -b reads.sorted.bam ...
```
However, in the `nanopolish variants` help section, `--consensus` doesn't take anything:
```
Usage: nanopolish variants [OPTIONS] --reads reads.fa --bam alignments.bam --genome genome.fa
Find SNPs using a signal-level HMM

  -v, --verbose                        display verbose output
      --version                        display version
      --help                           display this help and exit
      --snps                           only call SNPs
      --consensus                      run in consensus calling mode
      --fix-homopolymers               run the experimental homopolymer caller
  -w, --window=STR                     find variants in window STR (format: ctg:start-end)
  -r, --reads=FILE                     the 2D ONT reads are in fasta FILE
  -b, --bam=FILE                       the reads aligned to the reference genome are in bam FILE
```
 Is the help section just missing that part or is there something else going on? Is `polished.{1}.fa` the output for consensus?

**Comment by jts** [comment]
The help is missing that section, I will fix it. Thanks for pointing it out.

**Comment by igordot** [comment]
Thank you for the clarification. I wasn't sure if it was just missing or there was something I was not understanding.

**Comment by jts** [comment]
fixed in a1408a1
